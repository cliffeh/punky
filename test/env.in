(env)          ; ()
(define a 20)  ; a
(define b 30)  ; b
(env)          ; ((b . 30) (a . 20))
(keys (env))   ; (a)
