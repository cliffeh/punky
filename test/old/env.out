()
a
b
((b . 30) (a . 20))
(a b)
